src/Client/Im/Main.purs

```diff
@@ -727,7 +727,9 @@ toggleConnectedWebSocket isConnected model@{ hasTriedToConnectYet, errorMessage
             else
                   [ pure $ Just UpdateDelivered ]
       where
-      lostConnectionMessage = "Connection to the server lost. Attempting to automatically reconnect..."
+      lostConnectionMessage =
```

Comment:
This message is way too long for mobile screens. Lets dry it a bit maybe 

`Connection lost. Reconnecting...`